lightly/data/_video.py

```diff
@@ -283,6 +303,18 @@ def __init__(self,
         self.offsets = offsets
         self.fps = fps
 
+        # Current VideoLoader instance and the corresponding video index.
```

Comment:
Could you add a link to this PR and/or explain why we use this? So that whoever wants to change this code in the future knows why we implemented it like this.

Reply:
Added a comment with the link.